src/bfx_tools/isovar.ml

```diff
@@ -0,0 +1,66 @@
+open Biokepi_run_environment
+open Common
+
+module Configuration = struct
+
+  type t = {
+    name: string;
```

Comment:
No, it's the "unique" name of the configuration.

It's is often (always?) used to make unique paths (if you run the same tool with different parameters they won't step on each other).